### PR TITLE
Searchgenre

### PR DESCRIPTION
**Pull request by aminmus**
Opening PR to discuss changes and continue work on this.

### CODE REVIEW

File: movie-frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (aminmus) [inline movie-frontend/README.md]: I think this and related files are added by mistake maybe?

File: resources/views/selectedactor.blade.php [hunk 2]
```diff
@@ -1,6 +1,9 @@
 @extends('welcome')
 
 @section('content')
+<?php
+dd($movies)
```
Comment (aminmus) [inline resources/views/selectedactor.blade.php]: For debugging purposes I guess, but should be removed before merge.

File: resources/views/selectedfilm.blade.php [hunk 3]
```diff
@@ -19,6 +20,10 @@
       </div>
       <div class="card-body justify-content-center">
         <p>  {{$body->overview}}<p>
+        <p>  {{$body->vote_average}}<p>
```
Comment (aminmus) [inline resources/views/selectedfilm.blade.php]: Maybe we can add text or some kind of indicator here to clarify that what this is for the user.

File: app/Http/Controllers/SearchController.php [hunk 4]
```diff
@@ -17,13 +18,65 @@ public function __construct()
     }
 
     public function searchMovie(Request $request)
-    {
+    {   
+        
+        if ($request->searchoption === "genre") {
+            $genreId = '';
+            switch ($request->search) {
+                case "Action":
```
Comment (aminmus) [inline app/Http/Controllers/SearchController.php]: It seems the genre search is case sensitive, so when users search for a genre it has to be for example exactly "Action"  and not "action" or "ACTION". We can try making this case insensitive.

**Review by aminmus — COMMENTED** [review]
Is this relevant for this project or is it something added by mistake?

File: resources/views/selectedfilm.blade.php (hunk 3 above)
Comment (rubensalasamner) [reply resources/views/selectedfilm.blade.php]: Yet again animus saves the day. Very true, will rectify and add some sort of text to that explains the numbers

File: app/Http/Controllers/SearchController.php (hunk 4 above)
Comment (rubensalasamner) [reply app/Http/Controllers/SearchController.php]: Will look up workarounds, obviously the function should be flexible enough to allow a user to write capitalized as well as not

**Pull request merged**